manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0

```

```suggestion
PackageVersion: 1.3.0
```
Must match Apps&Features / Control Panel